experimental/fast_lossless/fast_lossless.cc

```diff
@@ -1297,7 +1299,9 @@ size_t LLEnc(const unsigned char* rgba, size_t width, size_t stride,
     PrepareDCGlobalPalette(onegroup, width, height, hcode, palette, pcolors,
                            &group_data[0][0]);
   }
+#ifdef __clang__
```

Comment:
I don't think this is the right #define - iirc gcc -fopenmp is a thing too.

Reply:
Changed to `#ifdef _OPENMP`.